components/ActivityCard.js

```diff
@@ -1,34 +1,46 @@
 import Image from "next/image";
+import Link from "next/link";
 import styled from "styled-components";
+import { useRouter } from "next/router";
 
-export default function ActivityCard({ title, categories, imageUrl }) {
-  return (
-    <StyledCardSection>
-      <StyledImageDiv>
+export default function ActivityCard({ id, title, categories, imageUrl }) {
+  const router = useRouter();
+
+  const handleClick = () => {
+    router.push(`/activity/${id}`);
+  };
+
+  return ( 
+    <StyledArticle>
+    <StyledImageContainer>
         <Image
           src={imageUrl}
           alt={title}
           style={{ objectFit: "cover" }}
           sizes="33vw"
           fill
         />
-      </StyledImageDiv>
-      <StyledList>
-        {categories.map((category) => (
-          <StyledListItem key={category}>{category}</StyledListItem>
-        ))}
-      </StyledList>
-      <StyledTitle>{title}</StyledTitle>
-    </StyledCardSection>
-  );
+    </StyledImageContainer>
+    <StyledList>
+      {categories.map((category) => (
+        <StyledListItem key={category}>{category}</StyledListItem>
+      ))}
+    </StyledList>
+    <StyledTitle>
```

Comment:
disclaimer: this is just a question from our side. Wouldn't you rather wrap the Link around the title and not the other way around ?

Reply:
We took another look and made some adjustments based on your feedback. Thank you for your input!